### DISCUSSION TITLE
error nova 4 Argument #1 ($mediaItem)

### DISCUSSION THREAD

**Discussion by MohammadAsafrahOyoun** [community/community#19759] [Other Feature Feedback, Questions, & Ideas]
App\\Models\\Complaint::Spatie\\MediaLibrary\\{closure}(): Argument #1 ($mediaItem) must be of type Spatie\\MediaLibrary\\MediaCollections\\Models\\Media, string given

**Comment by curtinaz** [comment]
You have to send more information. How can we get that error again?

**Comment by JhonnyLusonode** [comment]
Hello @MohammadAsafrahOyoun 
We need you to provide us more information about this error in order to help you out.
What is this error about and what steps did you do to get that error message?
